pyvista/core/filters/poly_data.py

```diff
@@ -2601,3 +2601,114 @@ def strip(poly_data, join=False, max_length=1000, pass_cell_data=False,
+            attribute named ``"ContactCells"``.  Array only exists
+            when there are collisions.
+
+        :class:`int`
+            Number of collisions.
+
+        Notes
+        -----
+        Due to the nature of the `vtk.vtkCollisionDetectionFilter
+        <https://vtk.org/doc/nightly/html/classvtkCollisionDetectionFilter.html>`_,
+        repeated uses of this method will be slower that using the
+        ``vtk.vtkCollisionDetectionFilter`` directly.  The first
+        update of the filter creates two instances of `vtkOBBTree
+        <https://vtk.org/doc/nightly/html/classvtkOBBTree.html>`_,
+        which can be subsequently updated by modifying the transform or
+        matrix of the input meshes.
+
+        This method assumes no transform and is easier to use for
+        single collision tests, but it is recommended to use a
+        combination of ``pyvista`` and ``vtk`` for rapidly computing
+        repeated collisions.  See the `Collision Detection Example
+        <https://kitware.github.io/vtk-examples/site/Python/Visualization/CollisionDetection/>`_
+
+        Examples
+        --------
+        Compute the collision between a sphere and the back faces of a
+        cube and output the cell indices of the first 10 collisions.
+
+        >>> import numpy as np
+        >>> import pyvista
+        >>> mesh_a = pyvista.Sphere(radius=0.5)
+        >>> mesh_b = pyvista.Cube((0.5, 0.5, 0.5)).extract_cells([0, 2, 4]).extract_surface()
+        >>> collision, ncol = mesh_a.collision(mesh_b, cell_tolerance=1)
+        >>> collision['ContactCells'][:10]
+        array([471, 468, 469, 466, 467, 464, 474, 475, 476, 477])
+
+        Plot the collisions by creating a collision mask with the
+        ``"ContactCells"`` field array.  Cells with a collision are
+        colored red.
+
+        >>> scalars = np.zeros(collision.n_cells, dtype=bool)
+        >>> scalars[collision.field_arrays['ContactCells']] = True
+        >>> pl = pyvista.Plotter()
+        >>> _ = pl.add_mesh(collision, scalars=scalars, show_scalar_bar=False, 
+        ...                 cmap='bwr')
+        >>> _ = pl.add_mesh(mesh_b, color='tan', line_width=5, opacity=0.7, 
+        ...                 show_edges=True)
+        >>> pl.show()
+
+        See Also
+        --------
+        See :ref:`collision_example` for more examples using this filter.
+
+        """
+        alg = _vtk.vtkCollisionDetectionFilter()
+        alg.SetInputData(0, poly_data)
+        alg.SetTransform(0, _vtk.vtkTransform())
+        alg.SetInputData(1, other_mesh)
+        alg.SetMatrix(1, _vtk.vtkMatrix4x4())
+        alg.SetBoxTolerance(box_tolerance)
+        alg.SetCellTolerance(cell_tolerance)
+        alg.SetNumberOfCellsPerNode(n_cells_per_node)
+        alg.SetCollisionMode(contact_mode)
+        alg.SetGenerateScalars(generate_scalars)
+        _update_alg(alg, progress_bar, 'Computing collisions')
+        return pyvista.wrap(alg.GetOutput()), alg.GetNumberOfContacts()
```

Comment:
Sorry, should have just used `_get_output`.

```suggestion
        return _get_output(alg), alg.GetNumberOfContacts()
```